addons/stock/models/stock_move.py

```diff
@@ -983,7 +989,7 @@ def _create_extra_move(self):
                 precision_rounding=self.product_uom.rounding,
                 rounding_method ='UP')
             extra_move_vals = self._prepare_extra_move_vals(extra_move_quantity)
-            extra_move = self.copy(default=extra_move_vals)._action_confirm()
+            extra_move = self.copy(default=extra_move_vals)._action_confirm(merge_into=self)
```

Comment:
only pass merge_into if picking id is set on self, add a comment that we don't want to merge extra move from mrp in v11 because it would change the behavior

note that we should probably do it in master very very soon before forgetting about it